ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -322,7 +323,7 @@ public struct Avatar: View, TokenizedControlView {
     }
 
     var contentSize: CGFloat {
-        return tokenSet[.avatarSize].float
+        return Self.avatarSize(state.size)
     }
```

Comment:
Do we still need contentSize?

Reply:
Yep, it's used by `AvatarGroup`, outside of `Avatar`.